code/modules/admin/verbs/adminpanelweapons.dm

```diff
@@ -97,6 +97,66 @@
 						addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(weaponhits), 2, picked_atom, point_defense), hiteta SECONDS)
 						message_admins("[key_name_admin(src)] Fired a single Railgun Slug at the Almayer at a random location, [picked_atom], with the possibility of missing as [point_defense]")
 
+		if("Orb. Def. Cannon")
+			if(exactplace == TRUE)
```

Comment:
you can just do `if(exactplace)` for truthy and `if(!exactplace)` for falsey

Reply:
:true: